File: 58-length-of-last-word/length-of-last-word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        n = len(s)

        count = 0

        for i in range(n-1,-1,-1):
            if count == 0 and s[i] == " " :
                continue
            if count > 0 and s[i] == " " :
                return count
            else:
                count+=1
        return count

File: 58-length-of-last-word/length-of-last-word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int n = s.size();
        int count=0;
        for(int i=n-1;i>=0;i--){

            if(s[i] == ' ' && count==0) continue;
            if(s[i] == ' ' & count>0) return count;
            else count++;
        }
        return count;
    }
};